### PR TITLE
Add DIVINE PCB to QMK

### PR DESCRIPTION
**Pull request by eerraa**
Add the ERA x DIVINE PCB to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/era/divine/info.json [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    "bootmagic": {
+        "matrix": [0, 1]
+    },
+    "indicators": {
+        "caps_lock": "GP1",
+		"num_lock": "GP2",
+        "scroll_lock": "GP3"
+    },
+    "layouts": {
+        "LAYOUT_split_all": {
```
Comment (dunk2k) [inline keyboards/era/divine/info.json:35]: - remove comma in last `features` element
- change LAYOUT name to conform to QMK layout naming convention
```suggestion
        "nkro": true
    },
    "matrix_pins": {
        "cols": ["GP13", "GP12", "GP19", "GP20", "GP21", "GP22", "GP23", "GP24", "GP25", "GP26", "GP27", "GP28", "GP29", "GP18", "GP5", "GP6", "GP7"],
        "rows": ["GP11", "GP10", "GP9", "GP8", "GP4"]
    },
    "processor": "RP2040",
    "url": "",
    "usb": {
        "device_version": "1.0.0",
        "pid": "0x0002",
        "vid": "0x4552"
    },
    "bootmagic": {
        "matrix": [0, 1]
    },
    "indicators": {
        "caps_lock": "GP1",
		"num_lock": "GP2",
        "scroll_lock": "GP3"
    },
    "layouts": {
        "LAYOUT": {
```

File: keyboards/era/divine/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS
+	),
+	
+	[5] = LAYOUT_split_all(
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
+				  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS
+	)
```
Comment (dunk2k) [inline keyboards/era/divine/keymaps/default/keymap.c:54]: - removed unused layers
- changed layout macro name to match `info.json` suggestion
```suggestion
    [0] = LAYOUT(
		KC_ESC,   KC_GRV,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_BSPC,  KC_INS,
		KC_PGUP,  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,               KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_DEL,
		KC_PGDN,  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,               KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,
				  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,  KC_UP,
				  KC_LCTL,                      KC_LALT,            KC_SPC,   MO(1),              KC_SPC,             KC_RALT,                      KC_LEFT,  KC_DOWN,  KC_RGHT
	),
	
    [1] = LAYOUT(
		KC_TRNS,  KC_TRNS,  KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_TRNS,  KC_TRNS,  KC_TRNS,
		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
				  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_VOLU,
				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_MPRV,  KC_VOLD,  KC_MNXT
	)
```

File: keyboards/era/divine/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS
+	),
+	
+	[5] = LAYOUT_split_all(
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
+				  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
+				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_TRNS,  KC_TRNS,  KC_TRNS
+	)
```
Comment (dunk2k) [inline keyboards/era/divine/keymaps/via/keymap.c:54]: - removed unused layers
- changed layout macro name to match `info.json` suggestion
- maximum layer count for VIA is 4, this change should correct CI Build failure
```suggestion
    [0] = LAYOUT(
		KC_ESC,   KC_GRV,   KC_1,     KC_2,     KC_3,     KC_4,     KC_5,     KC_6,     KC_7,     KC_8,     KC_9,     KC_0,     KC_MINS,  KC_EQL,   KC_BSPC,  KC_BSPC,  KC_INS,
		KC_PGUP,  KC_TAB,   KC_Q,     KC_W,     KC_E,     KC_R,     KC_T,               KC_Y,     KC_U,     KC_I,     KC_O,     KC_P,     KC_LBRC,  KC_RBRC,  KC_BSLS,  KC_DEL,
		KC_PGDN,  KC_CAPS,  KC_A,     KC_S,     KC_D,     KC_F,     KC_G,               KC_H,     KC_J,     KC_K,     KC_L,     KC_SCLN,  KC_QUOT,            KC_ENT,
				  KC_LSFT,  KC_NUBS,  KC_Z,     KC_X,     KC_C,     KC_V,     KC_B,     KC_B,     KC_N,     KC_M,     KC_COMM,  KC_DOT,   KC_SLSH,  KC_RSFT,  KC_UP,
				  KC_LCTL,                      KC_LALT,            KC_SPC,   MO(1),              KC_SPC,             KC_RALT,                      KC_LEFT,  KC_DOWN,  KC_RGHT
	),
	
    [1] = LAYOUT(
		KC_TRNS,  KC_TRNS,  KC_F1,    KC_F2,    KC_F3,    KC_F4,    KC_F5,    KC_F6,    KC_F7,    KC_F8,    KC_F9,    KC_F10,   KC_F11,   KC_F12,   KC_TRNS,  KC_TRNS,  KC_TRNS,
		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,
		KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,            KC_TRNS,
				  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_TRNS,  KC_VOLU,
				  KC_TRNS,                      KC_TRNS,            KC_TRNS,  KC_TRNS,            KC_TRNS,            KC_TRNS,                      KC_MPRV,  KC_VOLD,  KC_MNXT
	)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/era/divine/info.json [hunk 4]
```diff
@@ -0,0 +1,115 @@
+{
```
Comment (fauxpark) [inline keyboards/era/divine/info.json:1]: Please run `qmk format-json` on this file.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by dunk2k — APPROVED** [review]

File: keyboards/era/divine/info.json (hunk 4 above)
Comment (eerraa) [reply keyboards/era/divine/info.json:1]: > Please run `qmk format-json` on this file.

Sorry for the question.
What does it mean to run qmk format-json?

Comment (eerraa) [reply keyboards/era/divine/info.json:1]: Confirmed, Thank you sir.

Comment (eerraa) [reply keyboards/era/divine/info.json:1]: modified. Thanks for your help.

**Review by fauxpark — APPROVED** [review]

File: keyboards/era/divine/info.json [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "indicators": {
+        "caps_lock": "GP1",
+        "num_lock": "GP2",
+        "scroll_lock": "GP3",
+		"on_state": 0
```
Comment (nooges) [inline keyboards/era/divine/info.json:25]: Change tabs to spaces
```suggestion
        "on_state": 0
```

**Comment by eerraa** [comment]
@nooges Thank you sir!

**Review by fauxpark — CHANGES_REQUESTED** [review]
Keyboard updates need to go through `develop`.

**Comment by eerraa** [comment]
> Keyboard updates need to go through `develop`.

I don't know how to change the branch to develop. :(
I opened a new PR. I'll be careful.

**Comment by dunk2k** [comment]
> I don't know how to change the branch to develop. :( So i opened a new PR. I'll be careful.

[Change base branch of a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

**Comment by eerraa** [comment]
> > I don't know how to change the branch to develop. :( So i opened a new PR. I'll be careful.
> 
> [Change base branch of a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-base-branch-of-a-pull-request)

thank you I learned a lot thanks to you on how to use Github.
I have a question, I already wrote a new PR, should I revive this PR?
Or can I proceed with a new PR?

**Comment by dunk2k** [comment]
Depends on how confident you are with git rebase/merge.
If you can add the changes from `develop` branch to this branch then revival of this PR is feasible,
else proceed with new PR that's branched from`develop` branch 😉

**Comment by eerraa** [comment]
> Depends on how confident you are with git rebase/merge. If you can add the changes from `develop` branch to this branch then revival of this PR is feasible, else proceed with new PR that's branched from`develop` branch 😉
> 
> Edit: just seen the new PR for this board, best to remain with the new PR

@dunk2k Oh, Thank you very much always

**Pull request closed without merging**